src/mpyl/reporting/formatting/markdown.py

```diff
@@ -83,7 +83,7 @@ def run_result_to_markdown(run_result: RunResult) -> str:
     elif run_result.failed_result:
         failed = run_result.failed_result
         result += f"For _{failed.project.name}_ at _{failed.stage}_ \n"
-        result += f" > {run_result.failed_result.output.message} \n\n"
+        result += f"\n```\n{exception}\n```\n"
```

Comment:
We're in the `elif`, so exception is false here (test in line 80)? What content would it show?

Reply:
<img width="201" alt="Screenshot 2023-05-25 at 14 11 3👍 " src="https://github.com/Vandebron/mpyl/assets/1911436/176997cb-b941-4a6c-8f4f-65277b639a9f">
Indeed 👍 that's a bug

Reply:
Fixed in ef76663ddfe24f54572224a0d8fe7a745e49eb3f